durable-storage/src/persistence_layer.rs

```diff
@@ -176,4 +229,67 @@ mod tests {
                 .exists()
         );
     }
+
+    #[test]
+    fn test_basic_ops() {
+        let test = |value_a: String, value_b: String| {
+            let repo = DirectoryManager::new(std::path::Path::new("/tmp/test_basic_ops"))
```

Comment:
Nit: Why not use a `tempdir`-generated path here?

Reply:
This should make it easier to debug if the test actually fails

Reply:
it's in general not _super_ pleasant to ignore e.g. `TMPDIR` environment variable though? 

Why would it be harder to debug if using `tempdir`? (because the path would be cleaned automatically?)

If so, you _could_ have a wrapper over `tempdir` that only drops the path if we're [not panicking](https://doc.rust-lang.org/std/thread/fn.panicking.html) and otherwise if we are, print the path?